.gitignore

```diff
@@ -25,3 +25,4 @@ obj/
 .dub/
 libphobos_*
 source/
+/.project
```

Comment:
Why is this necessary?

Reply:
This is the project engineering document produced by my local IDE. It is not necessary to do so, but it will be more convenient for me to submit it it next time after doing so.

Reply:
It's okay to add, but please do so in the respective section of the gitignore and in a separate PR. Thanks!